src/main/java/racing/Car.java

```diff
@@ -1,5 +1,6 @@
 package racing;
```

Comment:
패키지 구조를 변경해보면 어떨까요?
자동차 경주의 모델들을 별도의 패키지로 이동하는 것과 안하는 것은 어떤 차이점이 있을까요?
```suggestion
package racing.domain;
```

Reply:
별도의 패키지는 추후 도메인 추가 시, 통일성 있게 관리하기 좋을듯 싶어요.
안하면, 많아지면 원하는 클래스를 찾기가 힘들 수도 있곘네요